benchmarks/opmphm.c

```diff
@@ -16,13 +19,18 @@
 #define NUMBEROFTHREADS 1
 #endif
 
-#define KDBRAND_BENCHMARK
-#include "../src/libs/elektra/opmphm.c"
-#include "../src/libs/elektra/rand.c"
+// ==== INCLUDE SECTION ====
 #include "benchmarks.h"
+#ifdef HAVE_HSEARCHR
+#include <search.h>
+#endif
+
 #ifdef USE_OPENMP
 #include <omp.h>
 #endif
+
+#include "../src/libs/elektra/opmphm.c"
```

Comment:
why is this needed?

Reply:
The mapping benchmarks uses the OPMPHM directly and the OPMPHM will not be exported.